### DISCUSSION TITLE
Erro de integração API

### DISCUSSION THREAD

**Discussion by demasnet** [RamonSilva20/mapos#1758] [General]
ola, o meu esta dando erro nos pagamento, na hora de faturar, fiz configurações para o asaas via API, e na hora da um erro (erro ao chamar o asaas)

**Comment by willph** [comment]
> ola, o meu esta dando erro nos pagamento, na hora de faturar, fiz configurações para o asaas via API, e na hora da um erro (erro ao chamar o asaas)

Você está usando em modo Sandbox ou produção?
